include/swift/Runtime/Metadata.h

```diff
@@ -834,6 +834,13 @@ void swift_enableDynamicReplacementScope(const DynamicReplacementScope *scope);
 SWIFT_RUNTIME_EXPORT
 void swift_disableDynamicReplacementScope(const DynamicReplacementScope *scope);
 
+// Shim to allow Swift to call metadata accessors.
+SWIFT_RUNTIME_EXPORT SWIFT_CC(swift)
```

Comment:
It might be more flexible to leave this with the C calling convention. That would make it easier for C-based libraries to also take advantage of this entry point.